amqp/platform.py

```diff
@@ -48,6 +48,8 @@ def _versionatom(s):
     if platform.release().endswith("Microsoft"):
         KNOWN_TCP_OPTS = {'TCP_NODELAY', 'TCP_KEEPIDLE', 'TCP_KEEPINTVL',
                           'TCP_KEEPCNT'}
+if sys.platform.startswith('darwin'):
```

Comment:
Maybe would be better to add this condition after the next if with an `elif`, like:

```
if sys.platform.startswith('linux'):
    [...]
elif sys.platform.startswith('win'):
    KNOWN_TCP_OPTS = {'TCP_NODELAY'}
elif sys.platform.startswith('darwin'):
    KNOWN_TCP_OPTS.remove('TCP_USER_TIMEOUT')
```

Otherwise, it would always execute both conditions: `if sys.platform.startswith('darwin'):` and `if sys.platform.startswith('darwin'):`.
Thanks for your PR!

Reply:
right, done 👍